partiql-ast/src/main/java/org/partiql/ast/v1/expr/ExprIon.java

```diff
@@ -0,0 +1,32 @@
+package org.partiql.ast.v1.expr;
+
+import org.jetbrains.annotations.NotNull;
+import org.partiql.ast.v1.AstNode;
+import org.partiql.ast.v1.AstVisitor;
+import com.amazon.ionelement.api.IonElement;
+
+import java.util.Collections;
+import java.util.Collection;
+
+/**
+ * TODO docs, equals, hashcode
+ */
+public class ExprIon extends Expr {
+    @NotNull
+    public IonElement value;
```

Comment:
This might actually be a good opportunity to just pass this as a string since this is just the AST. This could remove the only 3rd party dependency we have. Since it wouldn't be in the public API for the modeling, we could shadow IonReader in the evaluator to allow our customers to use whatever version of Ion they want in their own libraries.